int64 direction_x
int64 direction_y
int64 wheel_rotation

bool btn_right
bool btn_left
bool btn_middle